docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,231 @@
+
+ar71xx-tiny [#deprecated]_
+==========================
+
+* TP-Link
+
+  - TL-WR840N v2
+
+ipq40xx
+=======
+
+* 8devices
+
+  - Jalapeno
+
+mpc85xx-p1020 [#newtarget]_
+===========================
+
+* Aerohive
+
+  - HiveAP 330
+
+ramips-mt76x8
+=============
+
+* TP-Link
+
+  - TL-MR3420 v5
+
+ramips-mt7621
+=============
+
+* ASUS
+
+  - RT-AC57U [#noibss]
+
+.. [#deprecated]
+  This target will be reaching its end of life soon.
+
+.. [#newtarget]
+  This is a new target.
+
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
+
+.. note::
+
+    The ``sunxi`` target has been renamed to ``sunxi-cortexa7`` and ``ipq806x`` target has been flagged as broken, as all of its devices are not yet fully supported in this OpenWrt release. You might have to update your build scripts accordingly.
+
+
+
+New features
+************
+
+batman-adv coexistence
+======================
+
+To allow a migration from Batman 14, this Gluon release offers the ability to ship both B.A.T.M.A.N adv compat versions 14 and 15 in the same image. The ``mesh.batman_adv.routing_algo`` option decides which module gets loaded and the scheduled domain switching functionality can be used to migrate between the two versions.
+
+See the :ref:`mesh <user-site-mesh>` section for the *site.conf* configuration of this feature.                  
+
+Outdoor Mode
+============
+
+Radio devices hosted outdoors are often affected by different regulation than if they were installed indoors. The outdoor mode allows for the reconfiguration of 5 GHz radios on to different channels and channel ranges. Honoring stipulations like DFS currently make it next to impossible to use the 5 GHz band for mesh networking because it's never certain that nodes would stay on the same channel.
+If enabled, by setting ``wifi5.outdoor_chanlist``, a number of devices that are commonly installed outdoors will have outdoor mode automatically enabled during their initial setup, specifically:
+
+* Ubiquiti
+
+    - Bullet M
+    - Litebeam M5
+    - Nanostation M5
+    - Nanostation M5 Loco
+    - Rocket M5
+    - Rocket M5 TI
+    - Unifi AC Mesh
+    - Unifi AC Mesh Pro
+    - Unifi Outdoor
+
+* TP-Link
+
+    - CPE510
+    - WBS510
+
+See the :ref:`wifi5 <user-site-wifi5>` section for the *site.conf* configuration of this feature.                  
+
+
+Hoodselector: Geolocation Mode
+==============================
+
+Hoodselector is a new concept for continuous reevaluation of the nodes domain. As a first step we support its geolocation feature.
+
+See the :doc:`../package/gluon-hoodselector` documentation for details.
+
+
+Device Deprecation
+==================
+
+The ar71xx-tiny and several devices in the ramips-rt305x target have been marked as deprecated. The `GLUON_DEPRECATED` flag was introduced to offer downstream communities the choice on how to deal with the ending support for those devices.
```

Comment:
i think we should write a bit more concrete what "deprecated" and "ending support" means.
and for that, we need to agree on what it means, may be discussed on IRC

Reply:
The release notes reference this block in user/site.

```
GLUON_DEPRECATED

    Controls whether images for deprecated devices should be built. The following values are supported:

        0: Do not build any images for deprecated devices.
        upgrade: Only build sysupgrade images for deprecated devices.
        full: Build both sysupgrade and factory images for deprecated devices.

    Usually, devices are deprecated because their flash size is insufficient to support future Gluon versions. The recommended setting is 0 for new sites, and upgrade for existing configurations (where upgrades for existing deployments of low-flash devices are required).
```

Reply:
I'd like us to be more upfront in the release notes.
Something like "Devices or targets marked as deprecated are very likely to not be included in following Gluon releases anymore, usually due to their insufficient flash size."

Reply:
Agreed, appended wit minor modification.